client/app/lib/components/form/fields/DateTimePickerField.jsx

```diff
@@ -39,11 +39,13 @@ const FormDateTimePickerField = (props) => {
     disabled,
     label,
     renderIf,
+    required,
     style,
     className,
     variant = 'standard',
     disableMargins,
     disableShrinkingLabel,
+    suppressesFormatErrors,
```

Comment:
Because this means every time someone starts typing into an _empty_ `DateTimePickerField`, the field will immediately turn red, warning that the date/time format is invalid. That's counterproductive, since the date/time isn't wrong, but incomplete. Naturally, any errors should come _after_ form submission. After which, they can stay persistent and warn as the user types (fixes the problem). This is part of [the general good practices when building forms](https://www.nngroup.com/articles/errors-forms-design-guidelines/#:~:text=Don%E2%80%99t%20Validate%20Fields%20Before%20Input%20is%20Complete).

Note that `suppressesFormatErrors` does what it says, it suppresses _format errors_, i.e., that comes from the component itself. It does not suppress custom errors that were supplied to it, e.g., from react-hook-form.

Reply:
In such a case, should we default it to True?

Reply:
In an ideal world, yes. But, I am not sure of the current implementations in other parts of our app. As the usual mantra says, backwards compatibility.

If we default it to `true`, we can remove `suppressesFormatErrors` altogether. It should not cause any problems, but I did not do this the first time because when the error appears, it actually increases the height of the `DateTimePickerField`.